src/components/ForecastDetails.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/ForecastDetails.css";
+
+function ForecastDetails({ forecast }) {
+  const formattedDate = new Date(forecast.date).toDateString();
+  //   console.log(temperature.min, "<== min temperature");
+  // console.log(formattedDate);
+  // console.log(forecast.date);
+  return (
+    <div className="forecast-details">
+      <uo>
+        <div className="forecast-details__date"> {formattedDate}</div>
+        <div className="forecast-details__temperaturemax">
```

Comment:
i did write it like this, but for some reason something crashed the app,. It must have been something else though because it all works fine and tests are updated :)